pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2205,6 +2207,17 @@ Validation.Unique:
           default: UNIQUE
           enum:
             - UNIQUE
+Validation.UniqueValue:
+  allOf:
+    - $ref: '#/Validation'
+    - type: object
+      properties:
+        rule:
+          type: string
+          default: UNIQUE_VALUE
+          enum:
+            - UNIQUE_VALUE
+          description: check for uniqueness in same question answers among all participants of the study
```

Comment:
Let move this key up a few levels so it's at the same depth as the `properties` key above (the description will be associated with the object schema, and not the `rule` property)